libs/MotionKit/source/MotionKit.cpp

```diff
@@ -43,6 +43,11 @@ void MotionKit::rotate(uint8_t number_of_rotations, Rotation direction,
 	_motor_left.spin(direction, kPwmMaxValue);
 	_motor_right.spin(direction, kPwmMaxValue);
 
+	auto on_timeout = [this] { stop(); };
+
+	_timeout.onTimeout(on_timeout);
+	_timeout.start(10s);
```

Comment:
`_timeout.stop()` in `MotionKit::stop` too. Otherwise it motion will be too short for two Reinforcer in a row.